package.json

```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "format": "lerna run format",
```

Comment:
I would expect the `format` script to cover the root as well.

Reply:
The default "format" implementation is `prettier --write --loglevel warn \"./src/**/*.ts\"`, which does not do anything in the root dir. Thus I did not add it here.